### PR TITLE
Fixes a bunch of computers

### LINKED ISSUE #30615: Supply Console can't be set to allow contraband
In at least 2 shifts I've done today as quartermaster I've attempted to deconstruct a supply console and retool its board to "broad". However, when you put the board back in and reassemble the console, there is no contraband options because the board is somehow resetting itself once you put it back into the console.

To reproduce, disassemble a console, multitool the board to broad, reinsert and construct the machine

### PR DESCRIPTION
**Pull request by imtakingabreakdontatme**
:cl: Kor
fix: It is possible to multitool the cargo computer circuitboard for contraband again
/:cl:

Constructed computers will once again use the circuit they were built with which fixes cargo contraband as well as ID computers, the syndicate challenge lock, and the cooldown on spamming captain messages

Fixes #30615

### CODE REVIEW

File: code/game/machinery/computer/_computer.dm [hunk 1]
```diff
@@ -19,6 +19,10 @@
 /obj/machinery/computer/Initialize(mapload, obj/item/circuitboard/C)
 	. = ..()
 	power_change()
+	if(C)
+		qdel(circuit)
+		circuit = C
+		C.loc = src
```
Comment (ChangelingRain) [inline code/game/machinery/computer/_computer.dm]: use forceMove aaaaaa

File: code/game/machinery/computer/_computer.dm [hunk 2]
```diff
@@ -19,6 +19,10 @@
 /obj/machinery/computer/Initialize(mapload, obj/item/circuitboard/C)
 	. = ..()
 	power_change()
+	if(C)
```
Comment (ChangelingRain) [inline code/game/machinery/computer/_computer.dm]: if(!QDELETED(C))

**Review by Cyberboss — CHANGES_REQUESTED** [review]
Joan

**Review by Cyberboss — APPROVED** [review]

**Comment by lzimann** [comment]
i was busy and forgot about this, thanks

**Pull request merged**